datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxRepairService.java

```diff
@@ -76,15 +75,15 @@ public FlowIdentifier triggerRepairByName(String userCrn, String clusterName, Sd
         return sdxReactorFlowManager.triggerSdxRepairFlow(cluster.getId(), clusterRepairRequest);
     }
 
-    public void startSdxRepair(Long id, SdxRepairRequest repairRequest) {
+    public void startSdxRepair(Long id, SdxRepairSettings repairRequest) {
```

Comment:
Minor: rename `repairRequest` to `repairSettings`. Similarly in the other two changes below.